cmd/troubleshoot/cli/run.go

```diff
@@ -334,28 +335,35 @@ func loadClusterSpecs() (*troubleshootv1beta2.SupportBundle, *troubleshootv1beta
 
 	var bundlesFromCluster []string
 
+	parsedSelectorStrings, err := specs.SplitTroubleshootSecretLabelSelector(client, parsedSelector)
+	if err != nil {
+		klog.Errorf("failed to parse troubleshoot labels selector %s", err)
+	}
+
 	// Search cluster for support bundle specs
-	klog.V(1).Infof("Search support bundle specs from [%q] namespaces using %q selector", strings.Join(namespaces, ", "), parsedSelector.String())
-	for _, ns := range namespaces {
-		bundlesFromSecrets, err := specs.LoadFromSecretMatchingLabel(client, parsedSelector.String(), ns, specs.SupportBundleKey)
-		if err != nil {
-			if !k8serrors.IsForbidden(err) {
-				klog.Errorf("failed to load support bundle spec from secrets: %s", err)
-			} else {
-				klog.Warningf("Reading secrets from %q namespace forbidden", ns)
+	for _, parsedSelectorString := range parsedSelectorStrings {
+		for _, ns := range namespaces {
+			klog.V(1).Infof("Search support bundle specs from [%q] namespace using %q selector", strings.Join(namespaces, ", "), parsedSelectorString)
+			bundlesFromSecrets, err := specs.LoadFromSecretMatchingLabel(client, parsedSelectorString, ns, specs.SupportBundleKey)
```

Comment:
```suggestion
		klog.V(1).Infof("Search support bundle specs from [%q] namespace using %q selector", strings.Join(namespaces, ", "), parsedSelectorString)
		for _, ns := range namespaces {
			bundlesFromSecrets, err := specs.LoadFromSecretMatchingLabel(client, parsedSelectorString, ns, specs.SupportBundleKey)
```

Moving code out of for loop. The log statement logs all the namespaces (`[%q]`) being searched. No need of repeating the log line